src/js/content/npm.js

```diff
@@ -1,19 +1,29 @@
 /* global getBadge */
-const packageName = document.location.pathname.split('/')[2];
-const packageVersion = document.location.pathname.split('/')[4] || 'latest';
-const testPath = `https://snyk.io/test/npm/${packageName}/${packageVersion}`;
+function processNpmPackage() {
+  const packageName = document.location.pathname.split('/')[2];
+  const packageVersion = document.location.pathname.split('/')[4] || 'latest';
+  const testPath = `https://snyk.io/test/npm/${packageName}/${packageVersion}`;
 
-chrome.runtime.sendMessage({
-  source: 'npm',
-  packageName,
-  packageVersion,
-  testPath,
-}, () => {
-  const $anchor = document.createElement('a');
-  $anchor.setAttribute('href', `${testPath}`);
-  $anchor.innerHTML = getBadge(testPath);
+  chrome.runtime.sendMessage({
+    source: 'npm',
+    packageName,
+    packageVersion,
+    testPath,
+  }, () => {
+    const $anchor = document.createElement('a');
+    const $headingElement = document.querySelector('#readme h1') || document.querySelector('#readme > *:first-child');
```

Comment:
Fixes a small issue where projects without a h1 element in their README would cause an error. [Express is an example of this](https://www.npmjs.com/package/express)